data/credits/en.credits

```diff
@@ -1,6 +1,44 @@
 # Lines longer than 78 characters will be wrapped automatically
 # The following line is for reference
 ##############################################################################
+(<color_cyan>0.I</color>):</color>
+<color_white>Original author:</color>    <color_white>Project Manager:</color>
+<color_yellow>Whales</color> (<color_dark_gray>retired</color>)    <color_yellow>KevinGranade</color>
+
+<color_white>Current Main Developers:</color>
+<color_yellow>RenechCDDA</color> - Improvements to NPC factions, basecamps, and vehicles.
+
+<color_white>Special thanks to:</color>
+<color_yellow>Standing-Storm</color> - Creator and maintainer of the enormous 'Mind over Matter' mod.
+<color_yellow>Procyonae</color> - Highways!!!
+<color_yellow>PatrikLundell</color> - An absurd amount of bugfixes and important safety changes to mapgen, tripoint infrastructure, and other critical systems.
+<color_yellow>Anton Simakov 'GuardianDll'</color> - CDDA's newest merger, somehow they've also found the time to learn C++ and still be a big-time contributor.
+<color_yellow>RenechCDDA</color> - Improvements to NPC factions, basecamps, and vehicles.
+<color_yellow>Maleclypse</color> - Xedra Evolved maintainer and merger
```

Comment:
```suggestion
<color_yellow>Maleclypse</color> - Xedra Evolved maintainer and over 2k merges this cycle
```

Reply:
@Maleclypse I changed the wording a bit, check the commit and re-close this conversation if you don't have any issues with it.